ovl/kata/README.md

```diff
@@ -0,0 +1,52 @@
+# Xcluster/ovl - kata-containers
+
+kata-container ovl to include when needing to run microvm(s) in k8s
+
+## How to include in another ovl
+./kata.sh install <dir>
+
+## Kernel config needed to run kata
+These are the extra Kconfig needed on top of what is available in xcluster by default.
+'''
```

Comment:
Use back-ticks here!

There are _great_ plugins for firefox (Markdown viewer + github+mermaid) and Chrome (Markdown reader) that can be used to preview `.md` files.

Reply:
Ah, yes, will fix it. Thanks.